data/channels.csv

```diff
@@ -30155,7 +30155,7 @@ ZTVWorld.cn,ZTV World,,,CN,,,c/CN,zho,,FALSE,,,,,https://www.lyngsat.com/logo/tv
 ZuidWestTV.nl,ZuidWest TV,,,NL,,,c/NL,nld,,FALSE,,,,,https://i.imgur.com/SqslIN7.png
 ZukuNovella.ke,Zuku Novella,,,KE,,,c/KE,eng,,FALSE,,,,http://www.zuku.co.ke/,https://www.lyngsat.com/logo/tv/zz/zuku-novella-ke.png
 ZukuZone.ke,Zuku Zone,,,KE,,,c/KE,eng,,FALSE,,,,http://www.zuku.co.ke/,https://www.lyngsat.com/logo/tv/zz/zuku_zone.png
-ZuldemaydaComunicaciones.co,Zuldemayda Comunicaciones,,,CO,CO-QUI,Armenia,s/CO-QUI,spa,,FALSE,,,,http://zuldemaydacomunicaciones.com.co/,https://i.imgur.com/lR4DrKS.png
+ZuldemaydaCanal13.co,Zuldemayda Comunicaciones,,,CO,CO-QUI,Armenia,s/CO-QUI,spa,,FALSE,,,,http://zuldemaydacomunicaciones.com.co/,https://i.imgur.com/lR4DrKS.png

```

Comment:
```suggestion
ZuldemaydaCanal13.co,Zuldemayda Canal 13,,,CO,CO-QUI,Armenia,s/CO-QUI,spa,,FALSE,,,,http://zuldemaydacomunicaciones.com.co/,https://i.imgur.com/lR4DrKS.png
```

Reply:
Hmm I can't commit the suggestion through Github directly. I've gone ahead and apply the changes as requested.